### PR TITLE
alpine, apt*, dnf, ip*: add German translation

### PR DESCRIPTION
**Pull request by MarkusSYT**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.de/common/chmod.md
Comment (navarroaxel) [inline pages.de/common/chmod.md:33]: The translation should be made from the primary (English) pages.

If you want to add an extra example you should send a PR for edit the `pages/common/chmod.md`, and you can include the update the German translation for `pages.de/common/chmod.md` in that PR.

File: pages.de/common/docker.md
Comment (navarroaxel) [inline pages.de/common/docker.md:6]: The translation should be made from the primary (English) pages.

For docker we have specific subcommand pages like `docker-build.md`, `docker-start.md` etc.

If you want to add or edit examples you should send a PR for edit the primary page, and you can include the update for the German translation for the same page in that PR.

File: pages.de/common/docker.md
Comment (navarroaxel) [inline pages.de/common/docker.md:30]: Remember that the pages can have up to 8 examples, otherwise we can't merge the PR.

File: pages.de/linux/apt.md
Comment (navarroaxel) [inline pages.de/linux/apt.md:32]: This example is not present in the primary page.

File: pages.de/linux/apt.md
Comment (navarroaxel) [inline pages.de/linux/apt.md:52]: These examples are not present in the primary page. Remember that the pages can have up to 8 example and this have 12.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @MarkusSYT, welcome and thanks for your submission.

I'm sorry but your translations don't follow our rules completely, but don't worry. we can merge this if you:
1. Remove the `chmod.md` and `docker.md` pages from this PR.
2. Sync the `apt.md` and `apt-get.md` with the primary (English) pages (remove the examples that you have added).

Remember do not push force to the PRs because is harder for us to track the suggestion and changes, and feel free to open new PRs to add or edit the docker and chmod primary pages.

**Comment by MarkusSYT** [comment]
I changed it sorry for the trouble.

**Review by navarroaxel — APPROVED** [review]
Thanks @MarkusSYT, don't worry. I hope you'll send more PR!

**Comment by navarroaxel** [comment]
No @MarkusSYT :scream_cat:  rollback that push force, maybe using `git reflog`

**Comment by MarkusSYT** [comment]
Again sorry for that mistake :/
wasn't expecting that

File: pages.de/linux/ip-address.md
Comment (navarroaxel) [inline pages.de/linux/ip-address.md:19]: We use `snake_case` for variables:
```suggestion
`ip address add {{ip_adresse}} dev {{eth0}}`
```

File: pages.de/linux/ip-address.md
Comment (navarroaxel) [inline pages.de/linux/ip-address.md:23]:
```suggestion
`ip address delete {{ip_adresse}} dev {{eth0}}`
```

File: pages.de/linux/dnf.md
Comment (navarroaxel) [inline pages.de/linux/dnf.md:10]: Is there a translation for `keyword`?

File: pages.de/linux/ip.md
Comment (navarroaxel) [inline pages.de/linux/ip.md:31]: Are there translations for `mask` and `interface`?

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Don't worry @MarkusSYT. I just left some comments.

**Comment by MarkusSYT** [comment]
there would be a translation for mask and interface but it is not used in information technology

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @MarkusSYT!

**Comment by sbrl** [comment]
I'm going to  squash this to merge.

**Pull request merged**
